homeassistant/components/dyson/__init__.py

```diff
@@ -97,5 +97,6 @@ def setup(hass, config):
         discovery.load_platform(hass, "fan", DOMAIN, {}, config)
         discovery.load_platform(hass, "vacuum", DOMAIN, {}, config)
         discovery.load_platform(hass, "climate", DOMAIN, {}, config)
+        discovery.load_platform(hass, "air_quality", DOMAIN, {}, config)
```

Comment:
Looks like we can easily use a `for` loop here.

Reply:
Done.